Adding quick start entry for OpenShift (#110)
redhat-documentation/supplementary-style-guide: supplementary_style_guide/glossary_terms_conventions/product_conventions/openshift.adoc

*See also*: xref:action[action]

[discrete]
[[quick-start]]
==== quick start (noun)
*Description*: Two types of quick starts exist in OpenShift:

> * Quick starts that provide a guided tutorial in the web console

Reviewer: ```suggestion
* Quick starts that provide a guided tutorial in the web console.
```

Complete sentence so suggesting a period.

> * Quick start templates that enable users to start creating new applications quickly

Reviewer: ```suggestion
* Quick start templates that enable users to start creating new applications quickly.
```

> Be sure to provide context about which type of quick start you are referring to.

Reviewer: ```suggestion
Ensure that you provide context about which type of quick start you are referring to.
```
Suggestion to switch the phrasal verb out here. Feel free to ignore :)

*See also*: xref:action[action]

[discrete]
[[quick-start]]
==== quick start (noun)
*Description*: There are two types of quick starts in OpenShift:

> * quick starts that are guided tutorials in the web console

Author: Done!

> * Quick starts that are guided tutorials in the web console

Author: @dfitzmau I can't do a are/for swap here, so I tried to reword along the lines of how the next one is defined. Let me know if that's any better, or if you had a better suggestion for what you meant. Thanks!

```
Quick starts that provide a guided tutorial in the web console
```

> * Quick starts that are guided tutorials in the web console

Reviewer 2: HI @bergerhoffer . Thanks, LGTM.

> * Quick start templates that allow users to quickly get started creating a new application

Reviewer 3: s/allow/enable

Reviewer 4: enable or permit, instead of allow.
to quickly get started... hmm.  ... to start creating new applications quickly?

Author: Thanks, updated to enable and reworded some!
